baseplate/integration/pyramid.py

```diff
@@ -20,11 +20,11 @@ def make_app(app_config):
 .. warning::
 
     Because of how Baseplate instruments Pyramid, you should not make an
-    `exception view`_ that handles the base :py:exc:`Exception` as this will
-    prevent Baseplate from seeing the unhandled error and reporting it
-    appropriately.
+    `exception view`_ prevent Baseplate from seeing the unhandled error and
+    reporting it appropriately.
 
-    .. _exception view: https://docs.pylonsproject.org/projects/pyramid_cookbook/en/latest/pylons/exceptions.html#exception-views
+    .. _exception view: https://docs.pylonsproject.org/projects/pyramid_cookbook/\
```

Comment:
🔕 Kinda a bummer we can't click the link in the source anymore :(

Reply:
This was the only thing I could find that worked. There's no option in Sphinx to ignore instances of #noqa. I tried to find a different way of doing the notating the link but came up empty. I also looked for a way to tell flake8 to ignore it by adding something to a different line but nothing seemed to work.

Reply:
:( does this suggestion of putting the # noqa after the docstring work? https://stackoverflow.com/questions/10739843/how-should-i-format-a-long-url-in-a-python-comment-and-still-be-pep8-compliant/39768096#39768096

Reply:
No, unfortunately.